src/main/java/core/basesyntax/strategy/impl/ApplyBalance.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitNegotiation;
+import core.basesyntax.strategy.UnaryOperation;
+
+public class ApplyBalance implements UnaryOperation {
+    private FruitNegotiation myFruit;
+
+    public ApplyBalance(FruitNegotiation fruit) {
+        this.myFruit = fruit;
+    }
+
+    @Override
+    public void apply() {
+        Storage.storage.put(myFruit.getFruit(),myFruit.getQuantity());
```

Comment:
what about the remains from the previous day of if the supply has arrived in the middle of the day? do you just throw away the remains? Lets consider all possible situations

Reply:
refactored
but in task not mentioned about another day need more clarity task formulation